hack/release.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# This script builds and pushes a release to DockerHub.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+OS_ROOT=$(dirname "${BASH_SOURCE}")/..
+source "${OS_ROOT}/hack/common.sh"
+
+# Go to the top of the tree.
+cd "${OS_ROOT}"
+
+if [[ -z "${OS_TAG}" ]]; then
+  echo "You must specify the OS_TAG variable as the name of the tag to create, e.g. 'v1.0.1'."
+  exit 1
+fi
+tag="${OS_TAG}"
+
+if [[ "$(git name-rev --name-only --tags HEAD)" != "${tag}^0" ]]; then
+  if git rev-parse -q --short "${tag}" &>/dev/null; then
+    echo "Tag ${tag} already exists"
+    exit 1
+  else
+    git tag "${tag}" -a -m "${tag}" HEAD
+  fi
+fi
+
+function removeimage() {
+  for i in $@; do
+    if docker inspect $i &>/dev/null; then
+      docker rmi $i
+    fi
+    if docker inspect docker.io/$i &>/dev/null; then
+      docker rmi docker.io/$i
+    fi
+  done
+}
+
+removeimage openshift/origin centos openshift/base-centos7
```

Comment:
Isn't openshift/origin (the deployer) supposed to move to origin?  Removing openshift/base-centos7 seems appropriate since we'll need to grab the latest.